Makefile

```
@@ -0,0 +1,88 @@
+# 使用するシェルを明示
+# CI や他環境でデフォルトが異なると Make の挙動が変わるため zsh を指定する
+SHELL := /bin/zsh
+
+# .ONESHELL: 同一ターゲット内の複数行レシピを同じシェルで実行
+# 行間で cd や環境変数を共有する場合に有効
+# 問題なければ削除しても動作する
+.ONESHELL:
+
+# .PHONY: ターゲットをファイル名と区別して常に実行させる
+# 同名のファイルがあると Make がターゲットをスキップするのを防ぐ
+.PHONY: help dev db-up db-down db-status db-status_deprecated up up2 down down2 destroy destroy2 backend-dev frontend-dev install-backend install-frontend ci
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)